pkg/utils/watchChaosContainer.go

```diff
@@ -70,7 +70,7 @@ func GetChaosContainerStatus(experimentDetails *ExperimentDetails, clients Clien
 			Try(func(attempt uint) error {
 				pod, err := GetChaosPod(experimentDetails, clients)
 				if err != nil {
-					return errors.Errorf("Unable to get the chaos pod, error: %v", err)
+					return errors.Errorf("unable to get the chaos pod, error: %v", err)
```

Comment:
```suggestion
					return fmt.Errorf("unable to get the chaos pod, error: %w", err)
```

Reply:
we are using errors at every place. Will update in another PR